task/sanity-label-check/0.1/sanity-label-check.yaml

```diff
@@ -40,15 +40,24 @@ spec:
         --policy $(params.POLICY_DIR) --namespace $(params.POLICY_NAMESPACE) \
         --output=json 2> stderr.txt | tee sanity_label_check_output.json
 
-        ERR_MSG="$(cat stderr.txt)"
-        ERR_MSG="${ERR_MSG:-unknown}"
-        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --arg ERR_MSG "${ERR_MSG: 0: 3000}" --null-input \
-          '{result: "ERROR", timestamp: $date, failures: [$ERR_MSG]}')
+        if [ ! -z $(cat stderr.txt) ]; then
+          echo "The sanity-label-check test meets the following error:"
+          cat stderr.txt
+        fi
+        HACBS_ERROR_OUTPUT=$(jq -rc --arg date $(date +%s) --null-input '{result: "ERROR", timestamp: $date}')
         HACBS_TEST_OUTPUT=$(jq -rce --arg date $(date +%s) \
           '.[] | { result: (if (.failures | length > 0) then "FAILURE" else "SUCCESS" end),
                    timestamp: $date,
                    namespace,
                    successes,
-                   failures: (.failures // [])|map(.metadata.details.name) | unique
+                   failures: (.failures | length)
                  }' sanity_label_check_output.json || true)
+
+        # Log out the failing runs
+        if [ $(echo $HACBS_TEST_OUTPUT | jq '.failures') -gt 0 ]
+        then
+          echo "The sanity-label-check test fails with the following runs:"
+          echo $(jq '.[].failures' sanity_label_check_output.json)
```

Comment:
Maybe we can use the `.[].failures // []|map(.metadata.details.name)` jq filter here to display only the error names.